ts/session/apis/snode_api/getNetworkTime.ts

```diff
@@ -70,9 +70,16 @@ function getNowWithNetworkOffset() {
   return Date.now() - GetNetworkTime.getLatestTimestampOffset();
 }
 
+function getNowWithNetworkOffsetSeconds() {
```

Comment:
Did this get removed at some point? I felt like this was always here? It's on my onboarding branch

Reply:
probably just present on unstable, but not clearnet yet? I didn't find it anywhere